content/workshops/rhel_8/_index.adoc

```diff
@@ -0,0 +1,75 @@
+---
+title: Red Hat Enterprise Linux 8
+menu:
+  main:
+    parent: workshops
+    pre: fa fa-magic
+---
+
+:domain_name: redhatgov.io
+:workshop_prefix: workshop
+:tower_url: https://{workshop_prefix}.*student_number*.{domain_name}
+:ssh_url: https://{workshop_prefix}.*student_number*.{domain_name}:9090
+
+:icons: font
+:iconsdir: http://people.redhat.com/~jduncan/images/icons
+:imagesdir: /workshops/rhel_8/images
+
+= Red Hat Enterprise Linux 8 Workshop
+
+== Workshop Description
+
+This workshop will cover the features, specifications and improvements that comprise the release of version 8 of Red Hat Enterprise Linux (RHEL).  RHEL 8 brings many improvements over previous releases, including:
+
+* Application streams, which allow for the use of bleeding edge interpreters, compilers and libraries, while maintaining full backwards compatibility and stability of the system
+* The web console has been completely overhauled, and significant new features have been added
+** Session recording capability has been added to the web console
+* Image builder provides for the easy creation of customized RHEL installation images
+* The system package management tool, YUM, has been enhanced with DNF technology, providing significant performance improvements
+* System purpose has been enhanced to automatically tune system performance for the usage of the system
```

Comment:
Line 29 suggest: System features have been enhanced with automatic performance tuning for a better user experience.